features/actions.md

```diff
@@ -0,0 +1,31 @@
+# Claims
+
+Claims are the main way to create new tokens for your community. We use the concept of Objectives and Actions.
+
+### Objectives
+
+Objectives are what you want to achieve with your actions. All actions must be done towards a bigger objective. We recommend giving a through description to your objectives and try to add an expected result for it: a **good** result and a **great** result. This will help you get a better grasp on how your expectations are aligned with the reality.
+
+### Actions
+
+Actions are the definition of exactly what the community must do to reach those objectives. We made actions in a way that they are very flexible and allow for different use cases. For that we added a few options:
+
+**_Validity_ of an action. This can be by _date_ and _quantity_**
+  - _Date_: its pretty straightforward. The action can be claimed until a certain date. After that date is met, its no longer possible to open new claims. An important part is that the claims themselves can be validated after this date is met. Only new claims that cannot be created.
+  - _Quantity_ : Maybe we should have called this _usages_, but it determines the number of times this action can be **Approved or Disapproved**. That means that the users can claim all they want, but only the specified number can lead to rewards. So lets say we configure this action to have `Quantity of 3`; we can have 100s of claims opened, but after the 3 one is voted fully (approved or disapproved) the rest will be "unvoteable:, as the action will change its status to "completed" (that is set the flag `is_completed` to `true`.
+  
+**Verification**
+
+Verification is the process on which the community decides if the action was done or not. The verification can be *Automatic* if its going to be validated automatically by some software, or it can be *Manual*, which requires human validation.
```

Comment:
Here is my suggestion for this section:
Verification is the process by which the community decides if an action is completed or not. The verification process can be *Automatic* if validated by software, or *Manual*, which requires human validation.

Reply:
Ok!